docs/core-concepts/core-sdk/cli/dotnet-build.md

```diff
@@ -0,0 +1,85 @@
+dotnet-build
+===========
+
+## NAME 
+dotnet-build -- builds a project and all of its' dependencies 
+
+## SYNOPSIS
+
+dotnet-build [--output]  
+    [--build-base-path] [--framework]  
+    [--configuration]  [--runtime] [--version-suffix]
+    [--build-profile]  [--no-incremental] [--no-dependencies]
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-build` builds multiple source file from a source project and its dependencies into a binary. 
+The binary will be in Intermmidiate Language (IL) by default and will have a DLL extension. 
+`dotnet-build` will also drop a \*.deps file which outlines what the runner needs to run the application.  
+
+Building requires an existence of a lock file which means that a `dotnet-restore` call needs to happen 
```

Comment:
an existence -> the existence
Should you link to dotnet-restore if the user needs more info? 
I'd also prefer to transform this sentence in action item for the user so it's clear they need to do something about it. Something like: Building requires the existence of a `.lock` file, which means you have to run the `dotnet-restore` command before building your project.